File: ApiAndroid/ApiAndroid/Entities/Product.cs
namespace ApiAndroid.Entities
{
    public class Product
    {
        public int ProductId { get; set; }
        public string ProductName { get; set; } = null!;
        public string ProductNumber { get; set; } = null!;
        public virtual List<ProductPhoto> Photos { get; set; } = new List<ProductPhoto>();

        public virtual List<Document> Documents { get; set; } = new List<Document>();
    }
}

File: Productos/app/src/main/java/com/itcm/rodriguez/maria/productos/models/Product.kt
package com.itcm.rodriguez.maria.productos.models

data class Product(
    val productId: Int,
    val productName: String,
    val productNumber: String,
    val photos: List<ProductPhoto>,
    val documents: List<Document>
)

data class ProductDto(
    val productName: String,
    val productNumber: String,
    val photos: List<ProductPhoto>,
    val documents: List<Document>
)

data class ProductPhoto(
    val largePhoto: String,
    val productId: Int
)

data class Document(
    val documentId: Int = 0,
    val title: String,
    val fileName: String,
    val fileExtension: String,
    val revision: String,
    val documentFile: String,
    val productId: Int
)
